### DISCUSSION TITLE
Conda-forege package reports version number as '0.0.0'

### DISCUSSION THREAD

**Discussion by cnkym** [microsoft/Qcodes#4234] [Q&A] [answered]
After installed qcodes by "conda install -c conda-forge qcodes" on python 3.10.* or 3.9.*, 
the \_\_version\_\_ attribute will reply '0.0.0' if qcodes are 0.33.0 or 0.32.0. 
However it will get '0.31.0' if qcodes is 0.31.0.
> \>\>\> import qcodes
> \>\>\> qcodes.\_\_version\_\_
>'0.0.0'

Question : How to get the correct qcodes verion for 0.33.0 ?

**Comment by jenshnielsen** [comment]
There seems to be a bug in the way that conda generates versions from the pip version of the package. Our version managment has changed a bit since the last release but we will make sure this gets fixed with the next release

**Comment by jenshnielsen** [comment, marked as answer]
A fix has just been merged into the conda-forge feedstock. The new build (build 2 of 0.33) should be out in a few hours see https://github.com/conda-forge/qcodes-feedstock/pull/69

**Reply by cnkym** [reply]
I have confirmed the qcodes.\_\_version\_\_ shows '0.33.0' after "conda update -c conda-forge qcodes" to the latest 0.33.0. 
Thank you very much.
